system/lib/libc/musl/src/time/asctime.c

```diff
@@ -1,9 +1,9 @@
 #include <time.h>
 
-char *__asctime(const struct tm *, char *);
+char *__asctime_r(const struct tm *, char *);
 
 char *asctime(const struct tm *tm)
 {
 	static char buf[26];
-	return __asctime(tm, buf);
+	return __asctime_r(tm, buf);
 }
```

Comment:
this file is updated to latest upstream musl, the lack of `_r` was a bug apparently.

btw, this file is basically the same as the other non-reentrant ones - they allocate a buffer, and then call the reentrant version (which we implement in JS).